### PR TITLE
Add new fields to Application

### PR DESCRIPTION
**Pull request by quyenlq**

### CODE REVIEW

**Review by tuomas777 — COMMENTED** [review]
I think we'll want these to be `BooleanPGPPublicKeyField`s. Also, they should be nullable.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ repos:
     hooks:
     -   id: isort
 -   repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 22.3.0
```
Comment (tuomas777) [inline .pre-commit-config.yaml:17]: Nit: this change could be in its own commit.

**Review by tuomas777 — APPROVED** [review]
LGTM 👍

File: .pre-commit-config.yaml (hunk 1 above)
Comment (quyenlq) [reply .pre-commit-config.yaml]: This should belong to the other PR, I reverted this change
